definitions/divorce/json/CaseEvent/CaseEvent-bailiff-nonprod.json

```diff
@@ -75,7 +76,7 @@
     "Description": "Update Acknowledgement of Service",
     "DisplayOrder": 1,
     "PreConditionState(s)": "AosDrafted;AwaitingBailiffReferral;AwaitingBailiffService;IssuedToBailiff",
-    "PostConditionState": "AosDrafted",
+    "PostConditionState": "*",
```

Comment:
new callback not needed to handle change state here, as from each of the Precondition states it is supposed to go back to the same state

Reply:
Nice